Web/Edubase.Services.Texuna/Lookup/LookupApiService.cs

```diff
@@ -146,303 +146,303 @@ public async Task<IEnumerable<EstablishmentLookupDto>> EstablishmentTypesGetAllA
-        }
-        
-        public IEnumerable<LookupDto> PruEducatedByOthersGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> PruFulltimeProvisionsGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> PRUSENsGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> QualityAssuranceBodyNameGetAll()
-        {
-            throw new NotImplementedException();
-        }        
-        public IEnumerable<LookupDto> ReasonEstablishmentClosedGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> ReasonEstablishmentOpenedGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> ReligiousCharactersGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> ReligiousEthosGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> ResourcedProvisionsGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> Section41ApprovedGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> SpecialEducationNeedsGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> TeenageMothersProvisionsGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> TypeOfResourcedProvisionsGetAll()
-        {
-            throw new NotImplementedException();
-        }
-        
-        public IEnumerable<LookupDto> UrbanRuralGetAll()
-        {
-            throw new NotImplementedException();
-        }
-
-
-        #endregion
+        // #region Synchronous methods (deprecating)
+        //
+        // public IEnumerable<LookupDto> LocalAuthorityGetAll()
+        // {
+        //     throw new NotImplementedException();
+        // }
+        //
+        // public IEnumerable<LookupDto> AccommodationChangedGetAll()
+        // {
+        //     throw new NotImplementedException();
+        // }
+        //
+        // public IEnumerable<LookupDto> AdministrativeDistrictsGetAll()
+        // {
+        //     throw new NotImplementedException();
+        // }
+        //
+        // public IEnumerable<LookupDto> AdministrativeWardsGetAll()
+        // {
+        //     throw new NotImplementedException();
+        // }
+        //
+        // public IEnumerable<LookupDto> AdmissionsPoliciesGetAll()
+        // {
+        //     throw new NotImplementedException();
+        // }
```

Comment:
Better to delete these rather than just comment out?

Reply:
Yes I should

Reply:
Removed commented code